api-spec/wallet-api.yaml

```diff
@@ -245,8 +243,6 @@ paths:
       summary: Create new validation request given a initialized wallet
       description: This endpoint returns an object with a url to which to redirect in case of APM or a url on which to build an iframe for GDI check.
       operationId: postWalletValidations
-      security:
```

Comment:
same here

Reply:
I do not think it is useful to have it as we do not check the data. 
I think this attribute may be useful at the infrastructure level and not at the service level. What do you think?